### PR TITLE
UI/new buttons on guess location

### PR DESCRIPTION
**Pull request by BoostedBoat**
Next guess button now appears down right when confirm button has been pressed in guess location activity. It leads to the guess preview activity. The floating button to go to the scoreboard has been moved down left.

### CODE REVIEW

**Review by rkuenzi-epfl — CHANGES_REQUESTED** [review]
I think the new guess button should be a Floating Action Button or maybe extended FAB, see https://material.io/components/buttons-floating-action-button#extended-fab (under the extended FAB part). Though it might be @amattei777 jobs to change this.
Also as in your other PR you can just use VISIBLE instead of View.VISIBLE.

**Comment by amattei777** [comment]
> 
> 
> I think the new guess button should be a Floating Action Button or maybe extended FAB, see https://material.io/components/buttons-floating-action-button#extended-fab (under the extended FAB part). Though it might be @amattei777 jobs to change this.
> Also as in your other PR you can just use VISIBLE instead of View.VISIBLE.

Since I need to make all the button follow the same guide line for one of my task I would say that he doesn't need to worry to much about that yea. I will anyway have to look at it later this week so you can let me do that job.

File: app/src/main/res/anim/next_guess_button_anim.xml [hunk 1]
```diff
@@ -0,0 +1,10 @@
+<?xml version="1.0" encoding="utf-8"?>
+<set xmlns:android="http://schemas.android.com/apk/res/android"
+    android:fillAfter="true">
+
+    <alpha
+        android:fromAlpha="0.0"
+        android:toAlpha="1.0"
+        android:duration="1000" />
```
Comment (amattei777) [inline app/src/main/res/anim/next_guess_button_anim.xml:8]: This is the same alpha animation than for the score/distance text, I wonder if there is a way to factorise this. Maybe adding the values to strings.xml and then referancing them with "@string/..." would be better. I don't know if it is overkill tho.

**Review by amattei777 — CHANGES_REQUESTED** [review]
If the refactore remark seems overkill to you just tell me so that we can discuss it.

**Comment by BoostedBoat** [comment]
> I think the new guess button should be a Floating Action Button or maybe extended FAB, see https://material.io/components/buttons-floating-action-button#extended-fab (under the extended FAB part). Though it might be @amattei777 jobs to change this.
> Also as in your other PR you can just use VISIBLE instead of View.VISIBLE.

Actually, I made this choice because if I used a FAB, it would mean that in total, for the guess location activity, there would have been 4 different FABs. Even though only at most 2 of them would be displayed, I just thought it would be too confusing for the user to just have its FABs changing icons, and that a different button for going to a next activity (so an unrecoverable action) would be less confusing and more clear.

**Comment by BoostedBoat** [comment]
> I think the new guess button should be a Floating Action Button or maybe extended FAB, see https://material.io/components/buttons-floating-action-button#extended-fab (under the extended FAB part). Though it might be @amattei777 jobs to change this.
> Also as in your other PR you can just use VISIBLE instead of View.VISIBLE.

Also, I removed the View.VISIBLE and View.INVISIBLE

File: app/src/main/res/anim/next_guess_button_anim.xml (hunk 1 above)
Comment (BoostedBoat) [reply app/src/main/res/anim/next_guess_button_anim.xml:8]: done

**Review by rkuenzi-epfl — APPROVED** [review]
As discussed, if we choose to change the button it will be done by @amattei777 later so it's approved for me.

**Review by amattei777 — APPROVED** [review]

**Pull request merged**